virttest/tests/unattended_install.py

```diff
@@ -1157,6 +1157,38 @@ def string_in_serial_log(serial_log_file_path, string):
         return False
 
 
+def attempt_to_log_useful_files(test, vm):
+    """
+    Uses "vm.serial_console" to look in the system for useful log files
+    """
+    console = vm.serial_console
+    for path_glob in ["/*.log", "/tmp/*.log", "/var/tmp/*.log"]:
+        try:
+            paths = console.cmd("ls -1 %s" % path_glob)
+        except Exception as details:
```

Comment:
Do not need to assign the exception to a variable since you do nothing with it.

Reply:
sure, leftover from debugging...